Automattic/wp-calypso
# Smooth Loading Charts on GMB Stats Page

## Specs
Currently the chart placeholders on the Google My Business Stats Page are too different than the actual elements to ensure a smooth page load. The content will jerk and down as the data returns and charts load. This PR is to reduce this issue to a minimum.
## Testing
1. Navigate to the Google My Business stats page of a connected site on a local or live branch.
2. Verify that as the page loads it does not move up or down by more than a handful of pixels.
3. Number 2 does not apply to the very bottom. This issue is inherent to the way the D3Base component works and is out of scope for this PR
